eng/pipelines/templates/steps/analyze.yml

```diff
@@ -41,7 +41,7 @@ steps:
       Write-Host "##[command]Check source file formatting in $(pwd)"
       $files = Get-ChildItem -Path . -Filter *.go
       if ($files.Count -gt 0) {
-        gofmt -s -l -d $files
+        gofmt -s -l -d
```

Comment:
Do you still need lines 42 and 43 now? I assume `gofmt` will noop if there are no go files?

Reply:
I assumed it would do that too but I tested locally and that was not the case, it hangs if there's no .go files

Reply:
You need to pass in the directory as an argument to `gofmt` here (`gofmt -s -l -d .`) otherwise it will look for stdin as input instead. Passing in a directory argument does not hang when there are no go files, for me.